### ISSUE TITLE
Package with jlink, jdeps, jpackage

### ISSUE THREAD

**Issue by reid-spencer** [ossuminc/riddl#574]
It is possible since JDK 9 to get a more optimized JDK and runner generated with the various jxxx tools.  This should be doable and provide a small benefit, perhaps as the basis of a Brew tap. 
This video shows how:
https://www.youtube.com/watch?v=3UCBmdbeYm4

**Comment by reid-spencer** [comment]
This is blocked by Scala libraries which make use of sun.misc.Unsafe which lives outside the JDK 9 module system and therefore prevents execution of a reduce-by-jlink JDK (java.base only) because it throws a ClassNotFound exception. Consequently this needs to be on hold until https://github.com/scala/scala3/issues/9013 is fixed

**Comment by reid-spencer** [comment]
This capability will be released with 2.0
